package.json

```diff
@@ -6,7 +6,10 @@
   "homepage": "https://github.com/chanzuckerberg/edu-design-system",
   "license": "MIT",
   "main": "lib/index.js",
-  "sideEffects": false,
+  "sideEffects": [
+    "lib/tokens/css/variables.css",
+    "lib/tokens/fonts.css"
```

Comment:
What's happening on the Traject side without this?

When writing #1473, I didn't think this was necessary because Traject should be importing these explicitly.

Reply:
Fonts and CSS custom properties are being lost  
![image](https://user-images.githubusercontent.com/86632227/215880850-31e1360d-a9f3-4323-a303-777dc536517f.png)

Reply:
Dang, thanks for digging into this. Looks like you were totally right @anniehu4.

There's still something I don't understand here about how Webpack handles tree shaking (Jin and I confirmed that it does work as we expected in esbuild) 😓 

There are some old docs at https://github.com/webpack/webpack.js.org/pull/1865/files which mention marking css files as side effects. Other than that I haven't found much.